tutoraspects/plugin.py

```diff
@@ -136,6 +137,8 @@
         ("ASPECTS_EVENT_SINK_RECENT_BLOCKS_MV", "most_recent_course_blocks_mv"),
         # Vector settings
         ("ASPECTS_DOCKER_HOST_SOCK_PATH", "/var/run/docker.sock"),
+        ("ASPECTS_VECTOR_STORE_TRACKING_LOGS", False),
```

Comment:
These settings are new and let us turn on and off each piece of the Vector storage individually.